### PR TITLE
fix: remove unused net/url import from go

### PR DESCRIPTION
**Pull request by joeldodge79**
url.URL type properties were removed in v4 as of 21.8

fixes: #765

### CODE REVIEW

File: go/sdk/v4/models.go [hunk 1]
```diff
@@ -36,7 +36,6 @@ package v4
 
 import (
   "github.com/looker-open-source/sdk-codegen/go/rtl"
-  "net/url"
```
Comment (joeldodge79) [inline go/sdk/v4/models.go:39]: this is the direct cause of #765

File: package.json [hunk 2]
```diff
@@ -43,8 +43,8 @@
     "lint-quiet:es": "eslint 'packages/**/*.ts{,x}' --cache --quiet",
     "lint:ts": "tsc",
     "lint-staged": "yarn exec lint-staged",
-    "lint-changed": "eslint $(git diff --name-only --diff-filter=ACMRTUXB main | grep  -E \"(.js$|.ts$|.tsx$)\") --quiet",
-    "lint-changed-fix": "eslint $(git diff --name-only --diff-filter=ACMRTUXB main | grep  -E \"(.js$|.ts$|.tsx$)\") --quiet --fix",
+    "lint-changed": "eslint $(git diff --name-only --diff-filter=ACMRTUXB origin/main | grep  -E \"(.js$|.ts$|.tsx$)\") --quiet",
+    "lint-changed-fix": "eslint $(git diff --name-only --diff-filter=ACMRTUXB origin/main | grep  -E \"(.js$|.ts$|.tsx$)\") --quiet --fix",
```
Comment (joeldodge79) [inline package.json:47]: unrelated but commit hooks failed and I had to `lint-changed-fix` and I never keep a local `main` branch around so here we are

File: packages/sdk-codegen/src/go.gen.spec.ts [hunk 3]
```diff
@@ -34,6 +34,41 @@ const gen = new GoGen(apiTestModel)
 const indent = ''
 
 describe('Go generator', () => {
+  describe('bookends', () => {
+    it('has a v3 models prologue', () => {
+      const origPkgName = gen.packageName
+      gen.packageName = gen.apiVersion = 'v3'
+      const prologue = gen.modelsPrologue('')
+      gen.packageName = gen.apiVersion = origPkgName
```
Comment (joeldodge79) [inline packages/sdk-codegen/src/go.gen.spec.ts:42]: didn't really dig too deep into how `apiVersion` and `packageName` are set but this works

File: go/sdk/v4/models.go (hunk 1 above)
Comment (jkaster) [reply go/sdk/v4/models.go:39]: Ah. It doesn't load on demand like some of the other SDKs do. Good find/fix.

File: package.json (hunk 2 above)
Comment (jkaster) [reply package.json:47]: I think this is a better statement for the script anyway

**Review by jkaster — APPROVED** [review]
It's a reasonable stopgap fix. When we get back to working on this generator we should fix its import logic.

**Pull request merged**